### ISSUE TITLE
multi-line comments are losing linefeeds after edit/save

### ISSUE THREAD

**Issue by githubertus** [opensource-socialnetwork/opensource-socialnetwork#1443]
A comment like
Line 1
Line 2
appears like
Line 1 Line 2 ... some stuff added ...
after editing/saving

**Comment by githubertus** [comment]
Okay, so having a look at the old code
```
$text = $('#ossn-post-edit-form').find('#post-edit').val();
$guid = $('#ossn-post-edit-form').find('input[name="guid"]').val();
$elem = $("#activity-item-" + $guid).find('.post-contents').find('p:first');
if ($elem) {
      $elem.text($text);
}
```
my first idea was doing a
`$text = $text.replace(/\n/g,"<br>");`
but then we can't use the `$elem.text($text);` anymore.

On the other hand, using `$elem.html($text);` is no good idea, because people were able to insert any valid html code then.

So I ended up doing it like:
```
lines = $text.split('\n');
if ($elem) {
    $elem.text('');  // delete old stuff completely
    for(i = 0; i < lines.length; i++) {
	$elem.append(lines[i]);
	$elem.append("<br>");
    }
}
```

What do you think about it?

**Comment by githubertus** [comment]
Okay, so having a look at the old code

```
$text = $('#ossn-post-edit-form').find('#post-edit').val();
$guid = $('#ossn-post-edit-form').find('input[name="guid"]').val();
$elem = $("#activity-item-" + $guid).find('.post-contents').find('p:first');
if ($elem) {
      $elem.text($text);
}

```
my first idea was doing a
`$text = $text.replace(/\n/g,"<br>");`
but then we can't use the `$elem.text($text);` anymore.

On the other hand, using `$elem.html($text);` is no good idea, because people were able to insert any valid html code then.

So I ended up doing it like:

```
lines = $text.split('\n');
if ($elem) {
    $elem.text('');  // delete old stuff completely
    for(i = 0; i < lines.length; i++) {
	$(document.createTextNode(lines[i])).appendTo($elem);
	$elem.append("<br>");
    }
}

```
What do you think about it?

**Comment by lianglee** [comment]
Your solution is fine!

**Comment by githubertus** [comment]
fixed by #1450 and #1451
